modules/nf-core/gprofiler2/gost/meta.yml

```diff
@@ -0,0 +1,144 @@
+name: "gprofiler2_gost"
+description: runs a functional enrichment analysis with gprofiler2
+keywords:
+  - gene set analysis
+  - enrichment
+  - gprofiler2
+  - gost
+  - gene set
+tools:
+  - "gprofiler2":
+      description: "An R interface corresponding to the 2019 update of g:Profiler web tool."
+      homepage: "https://gl.cs.ut.ee/biit/r-gprofiler2"
+      documentation: "https://rdrr.io/cran/gprofiler2/"
+      tool_dev_url: "https://gl.cs.ut.ee/biit/r-gprofiler2"
+      doi: "10.1093/nar/gkad347"
+      licence: "['GPL v2']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing contrast information, e.g. [ variable:'treatment', reference:'treated', control:'saline', blocking:'' ]
+  - de_file:
+      type: file
+      pattern: "*.{csv,tsv}"
+      description: |
+        CSV or TSV-format tabular file with differential analysis outputs
+  - de_id_column:
```

Comment:
Someone else please confirm that we can write in meta arguments that are not in the main.nf inputs

Reply:
@sateeshperi Hey, should I then remove from the meta all arguments that are not channels?